### PR TITLE
fix: 🐛 add onWheelGuard to prevent unwanted preventDefault

### LINKED ISSUE #941: 启用scroller后默认的滚轮行为失效
这里无条件preventDefault导致启用scroller后默认的滚轮行为失效了

https://github.com/antvis/X6/blob/cec2c736ed79ebd810a48e13e480994dfdd439e8/packages/x6/src/util/dom/mousewheel.ts#L45

### PR DESCRIPTION
**Pull request by OpportunityLiu**
### Motivation and Context
fix #941

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Enhancement (changes that improvement of current feature or performance)
- [ ] Refactoring (changes that neither fixes a bug nor adds a feature)
- [ ] Test Case (changes that add missing tests or correct existing tests)
- [ ] Code style optimization (changes that do not affect the meaning of the code)
- [ ] Docs (changes that only update documentation)
- [ ] Chore (changes that don't modify src or test files)

### Self Check before Merge

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/antvis/x6/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by OpportunityLiu** [comment]
#943

**Comment by NewByVector** [comment]
@OpportunityLiu  是的，为了不影响线上项目，紧急发了一个版本，现在想用你的 PR，能重新提一下吗😶

File: packages/x6/src/util/dom/mousewheel.ts [hunk 1]
```diff
@@ -40,6 +43,10 @@ export class MouseWheelHandle {
   }
 
   private onWheel(e: JQueryMousewheel.JQueryMousewheelEventObject) {
+    if (this.onWheelGuard == null || !this.onWheelGuard(e)) {
```
Comment (OpportunityLiu) [inline packages/x6/src/util/dom/mousewheel.ts:46]: 这写错了，应该是 `this.onWheelGuard != null && !this.onWheelGuard(e)`

Comment (OpportunityLiu) [reply packages/x6/src/util/dom/mousewheel.ts:46]: Guard 为空应该相当于 `()=>true` 的

Comment (OpportunityLiu) [reply packages/x6/src/util/dom/mousewheel.ts:46]: @NewByVector

Comment (NewByVector) [reply packages/x6/src/util/dom/mousewheel.ts:46]: fixed

Comment (NewByVector) [reply packages/x6/src/util/dom/mousewheel.ts:46]: @OpportunityLiu  兄弟能留个联系方式吗

**Pull request merged**